examples/js/loaders/STLLoader.js

```diff
@@ -222,7 +222,7 @@ THREE.STLLoader.prototype = {
 
 			}
 
-			patternVertex = /vertex[\s]+([\-+]?[0-9]+\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]+([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]+([eE][\-+]?[0-9]+)?)+/g;
+			patternVertex = /vertex[\s]+([\-+]?[0-9]+\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]*([eE][\-+]?[0-9]+)?)+/g;
```

Comment:
1. It is clear that the regex could be more restrictive, but I guess this is not relevant in this case? On the other hand the regex should not be too restrictive, and it should not be different for the x coordinate and the other two.

2. This was only a question of mine  - I have no experience with STL, so I don't know whether there could be situations where the final `+` signs are used, though they are unnecessary in a file like your test.stl.

3. Now I saw that the simplification with {3} wouldn’t work, because the values of the first, third and fifth parentheses are used later in the code. Sorry, that I didn’t consider the code context yesterday.